cpp/src/centrality/katz_centrality.cu

```diff
@@ -34,10 +34,10 @@ void katz_centrality(Graph *graph,
                               double tol,
                               bool has_guess,
                               bool normalized) {
-  CUGRAPH_EXPECTS(graph->adjList != nullptr || graph->edgeList != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(graph->adjList->offsets->dtype == GDF_INT32, "Unsupported data type");
-  CUGRAPH_EXPECTS(graph->adjList->indices->dtype == GDF_INT32, "Unsupported data type");
-  CUGRAPH_EXPECTS(katz_centrality->dtype == GDF_FLOAT64, "Unsupported data type");
+  CHECK_GRAPH(graph);
+  CUGRAPH_EXPECTS(graph->adjList->offsets->dtype == GDF_INT32, "Unsupported data type: graph vertex IDs need to be int32");
```

Comment:
Are offsets vertex IDs? Shouldn't they be edge IDs?

Reply:
fixed